CMakeLists.txt

```diff
@@ -959,7 +973,9 @@ else()
 	set(PNG_REQUIRED_VERSION 1.6)
 endif()
 
+if(USE_SYSTEM_LIBPNG)
 find_package(PNG ${PNG_REQUIRED_VERSION})
```

Comment:
Can this be indented?

-[Unknown]

Reply:
Yes, this one is finding "system" libpng, which on macOS can come from a package manager, such as brew or MacPorts. Doing this will result in linking to a library that does not come with macOS and is thus broken.

Reply:
I just mean mechanically, the `find_package(PNG ${PNG_REQUIRED_VERSION})` should have a tab character before it.

-[Unknown]